src/rng.cpp

```diff
@@ -76,5 +76,6 @@ int djb2_hash( const unsigned char *str )
 double rng_normal( double lo, double hi )
 {
     static std::default_random_engine eng;
-    return std::normal_distribution<double>( ( hi + lo ) / 2, 3 )( eng );
+    double range = ( hi + lo );
+    return std::normal_distribution<double>( range / 2, range / 4 )( eng );
 }
```

Comment:
> This function is used in very few places.

That should probably change. Normal distribution is much better than linear random for many use cases.

> High and low are misnomers here, since you aren't even clamping the value to them

Will add clamping